TransIT.API/Controllers/EmployeeController.cs

```diff
@@ -110,6 +110,64 @@ public async Task<IActionResult> Create([FromBody] EmployeeDTO employeeDTO)
             }
         }
 
+        [HttpGet("boardnumber/{number}")]
+        public async Task<IActionResult> GetByBoardNumber(int number)
+        {
+            return Ok(await _employeeService.GetByBoardNumber(number));
+        }
+
+        [HttpGet("boardnumbers")]
+        public async Task<IActionResult> GetBoardNumbers()
+        {
+            return Ok(await _employeeService.GetBoardNumbers());
+        }
+
+        [HttpGet("attach/users")]
+        public async Task<IActionResult> GetNotAttachedUsers()
+        {
+            return Ok(await _employeeService.GetNotAttachedUsersAsync());
+        }
+
+        [HttpGet("attach/{userId}")]
+        public async Task<IActionResult> GetEmployeeForUserAsync([FromRoute] string userId)
+        {
+            return Ok(await _employeeService.GetEmployeeForUserAsync(userId));
+        }
+
+        [HttpPost("attach/{employee}/{user}")]
+        public async Task<IActionResult> AttachUserToEmployee([FromRoute] int employee, [FromRoute] string user)
+        {
+            var attachResult = await _employeeService.AttachUserAsync(employee, user);
+            IActionResult result;
+            if (attachResult == EmployeeDTO.CannotAttachUser)
+            {
+                result = BadRequest("Cannot attach this user, because it's already attached.");
+            }
+            else if (attachResult == EmployeeDTO.DoesNotExist)
+            {
+                result = BadRequest("User or employee doesn't exist.");
+            }
+            else
+            {
+                result = Ok(attachResult);
+            }
+            return result;
+        }
+
+        [HttpDelete("attach/{employee}")]
+        public async Task<IActionResult> RemoveUserFromEmployee([FromRoute] int employee)
+        {
+            var deleteResult = await _employeeService.RemoveUserAsync(employee);
+            if (deleteResult != EmployeeDTO.DoesNotExist)
+            {
+                return NoContent();
+            }
+            else
+            {
+                return BadRequest("Employee doesn't exist");
+            }
+        }
```

Comment:
add try catch for all endpoints, return 500 in case of exception, log the exception using logger

Reply:
@igolovchuk Can I use exception filter? It is already in the project in EndpointFilters folder

Reply:
yes. but check if it meets all requirements described earlier in comment